### ISSUE TITLE
Wtyczka Łączy Nas Pilka

### ISSUE THREAD

**Issue by walterek1** [mbebe/blomqvist#325]
Witam,
Czy jest może możliwość stworzenia wtyczki do Wideoteki ŁNP? https://www.laczynaspilka.pl/biblioteka/multimedia/index/wideoteka
Pozdrawiam!

**Comment by kashkash18** [comment]
Zainstaluj wtyczkę Ekstraklasa , w zakładce Kanały YT masz min Łączy nas piłka i Łączy nas piłka TV.
Musisz mieć zainstalowana wtyczkę YouTube.
Powinieneś tam znaleźć wszystko co Cię interesuje

**Comment by kashkash18** [comment]
Chociaż materiałów z Wideoteki nie ma w kanałach ŁNP i ŁNP TV.
A materiały są ciekawe :)
Może uda się namówić mbebe by dodał taka zakładkę do wtyczki Ekstraklasa.
Wymagać to będzie logowania w wtyczce gdyż wszystkie materiały są dostępne tylko po zalogowaniu.

**Comment by walterek1** [comment]
Dokładnie. Sprawdziłem to wcześniej, teraz również pogrzebałem i Wideoteki nie ma. :)

**Comment by kashkash18** [comment]
Wydaje mi się że nie ma sensu robienia osobnej wtyczki , ale do ekstraklasy można by to dodać :) @mbebe co Ty na to ? ;)

**Comment by mbebe** [comment]
Moze mi sie uda dzis puscic. ;-)

**Comment by mbebe** [comment]
Ok. dodane do wtyczki ekstraklasa.
Na ten moment odtwarzanie nie wymaga zalogowania.

**Comment by rafalmar1981** [comment]
wow ale fajnie dziękujemy

**Comment by walterek1** [comment]
Działa super.
Dzięki!

**Comment by kashkash18** [comment]
No i git.
Dzięki wielkie :-)
